lib/adapter.js

```diff
@@ -1750,8 +1749,8 @@ function Adapter(options) {
             let params = {};
             if (pattern && pattern !== '*') {
                 params = {
-                    startkey: pattern.replace('*', ''),
-                    endkey:   pattern.replace('*', '\u9999')
+                    startkey: pattern.replace(/\*/g, ''),

```

Comment:
I assume we want to replace all occurences, not only the first?

Reply:
I think the expectation is that there is maximum one ... ;-)

Reply:
@GermanBluefox

Reply:
Yes only one * is supported... So it is no matter, because with 2 will not work anyway